jobs/suite/default.env

```diff
@@ -1,2 +1 @@
-GINKGO_TEST_ARGS=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]
 GINKGO_PARALLEL=y
```

Comment:
Will do this in a separate PR (delete the suite/*.env files and migrate their configs to the job.env)